data/square-nurbs.mesh

```diff
@@ -0,0 +1,57 @@
+MFEM NURBS mesh v1.0
```

Comment:
This same file is duplicated in `miniapps/nurbs`

Reply:
I will delete the duplicate files in the miniapps/nurbs directory

Reply:
Removed